projects/fsastorefrontstyles/scss/components/cms/_unread-messages-indicator.scss

```diff
@@ -0,0 +1,32 @@
+cx-fs-unread-messages-indicator {
```

Comment:
3 `position: absolute;` declarations here (or anywhere for that metter) don't seem right nor optimal. One should be enough if the parent is relatively positioned.